linera-views/src/batch.rs

```diff
@@ -225,7 +225,27 @@ impl Batch {
         Ok(())
     }
 
-    /// Inserts the insertion of a `(key,value)` pair into the batch with value a vector of `u8`.
+    /// Adds the insertion of a `(key,value)` pair into the batch with a serializable value.
+    /// The obtained `Vec<u8>` is put into several `(key,value)` pairs if is larger than
```

Comment:
```suggestion
    /// The obtained `Vec<u8>` is put into several `(key,value)` pairs if it is larger than
```
(Or just "if larger than".)